src/app/(main)/page.tsx

```diff
@@ -0,0 +1,10 @@
+import type { Metadata } from "next"
+import { IndexPage } from "../../components/index-page"
+
+export const metadata: Metadata = {
+  title: "GraphQL | A query language for your API",
```

Comment:
oh I just noticed you already removed it from the index page, nice

here it is

```suggestion
  title: "GraphQL | A query language for your API",
  description: "GraphQL is a query language for APIs and a runtime for fulfilling those queries with your existing data."
```